quodlibet/formats/_id3.py

```diff
@@ -172,6 +172,8 @@ def __init__(self, filename):
             elif id3id == "USLT":
                 # lyrics are single string, not list
                 text = frame.text
+                self["~lyricsdescription"] = frame.desc
```

Comment:
Why `lyricsdescription` as the tag name? Is this a standard or an arbitrary choice?

Reply:
It was an arbitrary choice. In the standard it's called `content descriptor`, but there are content descriptors for several tags so there is no "official" unique name for it.